app/Http/Middleware/TrackRecentPages.php

```
@@ -0,0 +1,360 @@
+        UserRecentPage::recordVisit(
+            $user->id,
+            $team->id,
+            $path,
+            $labelData['label'],
+            $labelData['sublabel']
+        );
+
+        // Broadcast event to update all user's browser sessions (delay allows WebSocket to connect first)
+        $userId = $user->id;
+        dispatch(function () use ($userId) {
+            RecentsUpdated::dispatch($userId);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Events\RecentsUpdated' in method 'trackVisit'. (undefined)

(StaticAccess)